README.md

```diff
@@ -60,13 +61,13 @@ dataset/
 ```
 ## Getting started: data preprocessing
 
+### Dependencies for data preprocessing (see [dependencies](#dependencies_anchor))
+- [Spinal Cord Toolbox (SCT)](https://github.com/neuropoly/spinalcordtoolbox) version 5.8
```

Comment:
Re: commit 86ed4903dbbe5d830c45a06a238436244e75375a, I don't think you need to add anchor points? The markdown format (especially when displayed on GitHub) should naturally add anchor points you can reference (note the link icon next to section headers that you can hover over and click).

* ![image](https://github.com/neuropoly/template/assets/16181459/4cce6b34-1215-4357-b553-177d1d4f4cb8)

* ![image](https://github.com/neuropoly/template/assets/16181459/707106e1-17f8-4362-95b6-837ba9d43a8f)

So, `[dependencies](#dependencies)` _should_ work?